firmware/Makefile

```diff
@@ -56,6 +56,7 @@ USEMODULE += udp_m4a
 USEMODULE += serialization
 USEMODULE += at_commands_uart
 USEMODULE += system_init
+USEMODULE += storage
```

Comment:
this is not part of the module, i think it should not be here, what's you think about it?

Reply:
anyway modules should dynamically added so this will change in a near future